### DISCUSSION TITLE
i have one led for one touch

### DISCUSSION THREAD

**Discussion by maisonnavejul** [onlaj/Piano-LED-Visualizer#279] [Q&A]
hey,
I have a strip led WS2812B but with 60 leds. So i have one led for one touch. Who can i change code for not jumping leds ?

**Comment by onlaj** [comment]
There is currently no support for led strips other than 144 leds/m.
I will add this feature, possibly along with the option to custom align the leds with the keys. https://github.com/onlaj/Piano-LED-Visualizer/issues/270
No ETA yet.

**Reply by maisonnavejul** [reply]
okay, in my project i haven't need to adjuste i already have one led in front one touch. I juste want to remove the feature that adjusts

**Reply by onlaj** [reply]
You can edit it by yourself [here](https://github.com/onlaj/Piano-LED-Visualizer/blob/master/lib/functions.py#L241)
Keep in mind that "side keys" (lighting 3 leds with one key press) option will not work as intended with 60 leds/m.

**Reply by maisonnavejul** [reply]
yeah, i just want to show one song with one led for one touch. So if i change like that ? I just change to 2 to 0 and 1 to 0.  the programme will not jump leds ?
  if note > 92:
        note_offset = 0
    elif note > 55:
        note_offset = 0
    else:
        note_offset = 0

**Reply by onlaj** [reply]
This part is more important:
`note_pos_raw = 2 * (note - 20) - note_offset`
You have less than a half leds from normal setup, so you have to edit it to something like that:
`note_pos_raw = (note - 20) - note_offset`

**Reply by maisonnavejul** [reply]
Oh yeah of course, thank you
